fastlane/lanes/release.rb

```diff
@@ -284,6 +284,18 @@ def prompt_for_confirmation(message:, bypass:)
   UI.confirm(message)
 end
 
+def compute_release_branch_name(options:)
+  branch_option = :branch
+  branch_name = options[branch_option]
+
+  if branch_name.nil?
+    branch_name = release_branch_name
+    UI.message("No branch given via option '#{branch_option}'. Defaulting to #{branch_name}.")
+  end
+
+  branch_name
+end
```

Comment:
I did this because the history in my shell had `branch_name` as the option name.

Maybe it's over-engineered given we use it only every once in a while. Still, I wish this change had been there when I needed it as it would have made it easier to understand how to get the result I hoped for.